test/e2e/mv3/multiple-restarts.spec.js

```diff
@@ -0,0 +1,294 @@
+const { strict: assert } = require('assert');
+const {
+  withFixtures,
+  openDapp,
+  generateGanacheOptions,
+  WALLET_PASSWORD,
+} = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+
+const WINDOW_TITLES = Object.freeze({
```

Comment:
Suggestion: i can see some of them are used in other tests, we could take advantage of this to move the enum to helper and have future tickets to refactor the usage of that . wdyt?